halo2_gadgets/src/ecc/chip/mul_fixed/short.rs

```diff
@@ -519,39 +519,39 @@ pub mod tests {
                 // 2^64
                 MyCircuit {
                     magnitude: Value::known(pallas::Base::from_u128(1 << 64)),
-                    sign: Value::known(pallas::Base::one()),
+                    sign: FieldValue::ONE,
```

Comment:
Similarly here (though the length difference could be smaller as `pallas::Base::ONE` is now also valid).